mypy/checker.py

```diff
@@ -2280,6 +2280,25 @@ def check_method_override_for_base_with_name(
             if inner is not None:
                 typ = inner
                 typ = get_property_type(typ)
+
+                if (
+                    isinstance(original_node, Var)
+                    and original_node.is_classvar
+                    and defn.name == original_node.name
+                    and (isinstance(defn, (Decorator, OverloadedFuncDef)))
+                ):
+                    decorator_func = None
+                    if isinstance(defn, Decorator):
+                        decorator_func = defn.func
+                    elif isinstance(defn.items[0], Decorator):
+                        decorator_func = defn.items[0].func
+
+                    if decorator_func:
+                        self.fail(
+                            message_registry.CANNOT_OVERRIDE_CLASS_VAR.format(base.name),
+                            decorator_func,
+                        )
```

Comment:
wondering if the error code `OVERRIDE` should be attached to this error message? left it as `misc` to keep behaviour consistent with existing error message:

https://github.com/python/mypy/blob/f2f8a98e10e5b6d794d338aaf8edce01e3206a4f/mypy/checker.py#L3722

Reply:
No, `override` code is for method/attribute overrides (LSP violation), `misc` looks reasonable